designer/server/src/routes/forms/create.js

```diff
@@ -13,6 +13,15 @@ import { buildErrorDetails } from '~/src/common/helpers/build-error-details.js'
 import * as forms from '~/src/lib/forms.js'
 import * as create from '~/src/models/forms/create.js'
 
+/**
+ * Gets the author from the auth credentials
+ * @param {import('@hapi/hapi').UserCredentials | undefined} user
```

Comment:
Mind keeping these in the footer?

We'll swap them all with `@import` blocks soon

Can use ~`user`~ `[user]` to handle potentially `undefined`

```suggestion
 * @param {UserCredentials} [user]
```